opal-core/src/main/java/org/obiba/opal/core/service/VCFSamplesMappingServiceImpl.java

```diff
@@ -68,9 +70,11 @@ public void delete(@NotNull String projectName) throws NoSuchVCFSamplesMappingEx
   }
 
   @Override
-  public List<String> getFilteredSampleIds(@NotNull String projectName, @NotNull String filteringTable, boolean withControl) {
-    ValueTable filteringValueTable = MagmaEngineTableResolver.valueOf(filteringTable).resolveTable();
-    List<String> participantIds = filteringValueTable.getVariableEntities().stream().map(VariableEntity::getIdentifier).collect(Collectors.toList());
+  public List<String> getFilteredSampleIds(@NotNull String projectName, String filteringTable, boolean withControl) {
+    List<String> participantIds = Strings.isNullOrEmpty(filteringTable) ? 
+        Lists.newArrayList() :
```

Comment:
no filteringTable => samples of all participants + orphane samples

I don't think this is the case with your code.